include/PianoView.h

```diff
@@ -45,9 +45,9 @@ class PianoView : public QWidget, public ModelView
 public:
 	PianoView( QWidget * _parent );
 	~PianoView() override = default;
-
```

Comment:
I personally prefer having a space between constructors/destructors and the rest of the members, but I don't think we have a strict rule about this.

Reply:
Btw shouldn't the destructor be virtual here (even if that wasn't part of this PR)?

Reply:
It already is, hence the `override`. It's preferred not to specify `virtual` along with `override`: see [the entry in the Core Guidelines](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rh-override).